include/rocksdb/options.h

```diff
@@ -1479,6 +1479,15 @@ struct IngestExternalFileOptions {
   // Warning: setting this to true causes slowdown in file ingestion because
   // the external SST file has to be read.
   bool verify_checksums_before_ingest = false;
+  // Set to true if you would like to ingest SST files generated by live DB.
+  // Notice: because SST files generated by live DB don't has global_seq table
+  // property, so you must set write_global_seqno to false.
+  bool ingest_live_db_files = false;
+  // Set to false if you would like to ingest SST files with different CF ID
+  // table property.
+  // Warning: set it to false maybe make RepairDB useless after MANIFEST
+  // corruption.
+  bool check_cf_id_property_before_ingest = true;
```

Comment:
Name "check" is ambiguous. We should say "allow" or not.

Reply:
'allow_cf_id_property_check_before_ingest' ?

Reply:
'allow_ingest_files_with_different_cf_id_property' ?

Reply:
'allow_ingest_files_with_different_cf_id_property' sounds better to me.